src/main/resources/db/test/migration/R__Initial_data.sql

```diff
@@ -81,45 +82,62 @@ values (4, 'Test collection #1 of biobank #1', 'This is the first test collectio
+       ('negotiation-v2', '2023-04-12', 'ABANDONED', '2024-10-12', 108, 108,
+        '{"project":{"title":"A Project 3","description":"Project 3 desc"},"samples":{"sample-type":"Blood","num-of-subjects": 5,"num-of-sample": "10","volume":4},"ethics-vote":{"ethics-vote":"My ethics"}}' FORMAT JSON),
+       ('negotiation-3', '2024-02-24', 'IN_PROGRESS', '2024-02-24', 105, 105,
+        '{"project":{"title":"Project 3","description":"Project 3 desc"},"samples":{"sample-type":"Blood","num-of-subjects": 5,"num-of-sample": "10","volume":4},"ethics-vote":{"ethics-vote":"My ethics"}}' FORMAT JSON),
+       ('negotiation-4', '2024-01-10', 'ABANDONED', '2024-01-10', 105, 105,
         '{"project":{"title":"Project 3","description":"Project 3 desc"},"samples":{"sample-type":"Blood","num-of-subjects": 5,"num-of-sample": "10","volume":4},"ethics-vote":{"ethics-vote":"My ethics"}}' FORMAT JSON);
 
 insert into request (id, url, human_readable, data_source_id, negotiation_id)
 values ('request-1', 'http://datasource.dev', '#1: No filters used', 1, 'negotiation-1'),
        ('request-2', 'http://datasource.dev', '#1: DNA Samples', 1, null),
        ('request-v2', 'http://datasource.dev', '#1: Blood Samples', 1, 'negotiation-v2'),
+       ('request-3', 'http://datasource.dev', '#1: Blood Samples', 1, 'negotiation-3'),
+       ('request-4', 'http://datasource.dev', '#1: Blood Samples', 1, 'negotiation-4'),
        ('request-unassigned', 'http://datasource.dev', '#1: Blood Samples', 1, null);
 
 insert into request_resources_link (request_id, resource_id)
 values ('request-1', 4),
        ('request-2', 5),
        ('request-2', 6),
        ('request-v2', 7),
+       ('request-3', 5),
+       ('request-4', 5),
        ('request-unassigned', 7);
 
 insert into resource_state_per_negotiation (negotiation_id, resource_id, current_state)
 values ('negotiation-1', 'biobank:1:collection:1', 'SUBMITTED'),
        ('negotiation-1', 'biobank:1:collection:2', 'SUBMITTED'),
        ('negotiation-v2', 'biobank:3:collection:1', 'SUBMITTED');
 
-insert into person_negotiation_role (negotiation_id, person_id, role_id)
-values ('negotiation-1', 108, 4);
+-- Adding just representatives since the AUTHOR is the createdBy field of the negotiation
 insert into person_negotiation_role (negotiation_id, person_id, role_id)
 values ('negotiation-1', 109, 5);
 insert into person_negotiation_role (negotiation_id, person_id, role_id)
-values ('negotiation-2', 108, 4);
+values ('negotiation-2', 109, 5);
+insert into person_negotiation_role (negotiation_id, person_id, role_id)
+values ('negotiation-v2', 109, 5);
+insert into person_negotiation_role (negotiation_id, person_id, role_id)
+values ('negotiation-3', 105, 5);
 insert into person_negotiation_role (negotiation_id, person_id, role_id)
-values ('negotiation-v2', 108, 4);
+values ('negotiation-4', 105, 5);
```

Comment:
this is no longer used. The representatives are handled via the resourc_rep_link

Reply:
I had to keep one line because some tests using httpBasic auth still need it. We should get rid of httpBasic in the future and clean unused entities.